### ISSUE TITLE
Feature request: filter rest queries by id in `one.alyx.rest`

### ISSUE THREAD

**Issue by mayofaulkner** [int-brain-lab/iblenv#63]
**Describe the bug**
When trying to access insertions endpoint using the probe insertion id an extra `/` is required

**To Reproduce**
```
insertion = one.alyx.rest('insertions', 'list', id='1f7a73da-d42c-4b1a-a999-b3ddeead6159')
```
gives the error
```
2020-10-02 11:32:21.359 ERROR    [webclient.py:248] https://alyx.internationalbrainlab.org/insertions1f7a73da-d42c-4b1a-a999-b3ddeead6159
2020-10-02 11:32:21.362 ERROR    [webclient.py:249] <h1>Not Found</h1><p>The requested resource was not found on this server.</p>
```

However
```
insertion = one.alyx.rest('insertions', 'list', id='/1f7a73da-d42c-4b1a-a999-b3ddeead6159')
```
returns the insertion object without error

**Expected behavior**
Expect behaviour to be 
```
insertion = one.alyx.rest('insertions', 'list', id='1f7a73da-d42c-4b1a-a999-b3ddeead6159')
```
rather than
```
insertion = one.alyx.rest('insertions', 'list', id='/1f7a73da-d42c-4b1a-a999-b3ddeead6159')
```

**Comment by oliche** [comment]
I think the recommended usage is to use the read method when providing a single id:
`one.alyx.rest('insertions', 'read', id='1f7a73da-d42c-4b1a-a999-b3ddeead6159')`

The reason this happens is because there is a collision between the `id` model field and the `id` parameter of the `one.alyx.rest` method.

**Comment by oliche** [comment]
Fixed in ibllib develop 777510d6a89471c195c6bde895d030a5c9ee7ce7
